src/token.ts

```diff
@@ -39,8 +39,28 @@ function getAccountBalanceForTokenContractAddress(
   )
 }
 
+function getHistoricAccountBalanceForTokenContractAddress(
```

Comment:
I'd rename this as `getHistoricialAccountBalanceForTokenContractAddress` to be consistent with the docs